app/helpers/argo_helper.rb

```diff
@@ -104,6 +104,9 @@ def render_buttons(doc, object = nil)
       if object.datastreams.include? 'rightsMetadata'
         buttons << {:url => url_for(:controller => :items, :action => :rights, :id => pid), :label => 'Set rights'}
       end
+      if object.datastreams.include?('identityMetadata') && object.identityMetadata.otherId('catkey').present? # indicates there's a symphony record
+        buttons << {url: url_for(controller: :items, action: :refresh_metadata, id: pid), label: 'Refresh descMetadata', new_page: true}
```

Comment:
I think it's preferable to use (or create) a named route (allowing us to refactor things like controller and action names without breaking compatibility). Is that possible here?